pkg/apis/seedmanagement/v1alpha1/defaults_managedseed.go

```diff
@@ -116,7 +116,19 @@ func setDefaultsGardenletConfiguration(obj *gardenletv1alpha1.GardenletConfigura
 
 	// Initialize seed config
 	if obj.SeedConfig == nil {
-		obj.SeedConfig = &gardenletv1alpha1.SeedConfig{}
+		obj.SeedConfig = &gardenletv1alpha1.SeedConfig{
+			SeedTemplate: gardencorev1beta1.SeedTemplate{
+				Spec: gardencorev1beta1.SeedSpec{},
+			},
+		}
+	}
+
+	if obj.SeedConfig.SeedTemplate.Spec.Ingress == nil {
+		obj.SeedConfig.SeedTemplate.Spec.Ingress = &gardencorev1beta1.Ingress{
+			Controller: gardencorev1beta1.IngressController{
+				Kind: "nginx",
+			},
+		}
```

Comment:
I would expect to see a test that verifies that you are now able to create a ManagedSeed without specifying `.spec.gardenlet` (the real motivation for the issue this PR is fixing). If we address the previous comment, then we can add it as an unit test to the admission plugin.